### PR TITLE
Add support for message of an non, recognized command

### PR DESCRIPTION
**Pull request by Qlenglet**
Add the option to send a message if a command is not recognized. The schema of the database must be change too.

Thx

### CODE REVIEW

File: Quarterdeck/src/main/java/com/fredboat/backend/quarterdeck/rest/v1/transfer/GuildConfig.java [hunk 1]
```diff
@@ -75,4 +77,6 @@ public boolean isClearOnEmpty() {
     public Language getLanguage() {
         return this.language;
     }
+
+    public boolean isEnableUnknownCommand() { return this.enableUnknownCommand;}
```
Comment (napstr) [inline Quarterdeck/src/main/java/com/fredboat/backend/quarterdeck/rest/v1/transfer/GuildConfig.java]: Missing whitespace

File: Quarterdeck/src/main/java/com/fredboat/backend/quarterdeck/db/migrations/main/V1__InitialSchema.java [hunk 2]
```diff
@@ -75,6 +75,7 @@ public void migrate(Context context) throws Exception {
                 + "    track_announce BOOLEAN NOT NULL, "
                 + "    auto_resume    BOOLEAN NOT NULL, "
                 + "    lang           CHARACTER VARYING(255) COLLATE pg_catalog.\"default\" NOT NULL, "
+                + "    msg_unknown_command BOOLEAN NOT NULL"
```
Comment (napstr) [inline Quarterdeck/src/main/java/com/fredboat/backend/quarterdeck/db/migrations/main/V1__InitialSchema.java]: Please read up on how Flyway works. Manipulating existing migrations is not the correct way to achieve what you want.

Also the column name here does not correspond to the column name used in the entity.

Comment (Qlenglet) [reply Quarterdeck/src/main/java/com/fredboat/backend/quarterdeck/db/migrations/main/V1__InitialSchema.java]: Hi, thank you for your return but I don't understand how I can do what you're asking for.
Thank you again for your cooperation and help.
I send an update if you want to see.

Comment (napstr) [reply Quarterdeck/src/main/java/com/fredboat/backend/quarterdeck/db/migrations/main/V1__InitialSchema.java]: Can you be more specific? Which part of my message are you having trouble with understanding?

Comment (Qlenglet) [reply Quarterdeck/src/main/java/com/fredboat/backend/quarterdeck/db/migrations/main/V1__InitialSchema.java]: About the manipulation of already existing migration.
Can you see the last update ?

Thank you a lot

Comment (napstr) [reply Quarterdeck/src/main/java/com/fredboat/backend/quarterdeck/db/migrations/main/V1__InitialSchema.java]: > About the manipulation of already existing migration.

Please read more closely what I have said (and follow it):
> Please read up on how Flyway works.

This PR is not the right place to teach how Flyway works and also out of scope of things that I am willing to do. If you are not willing to learn about Flyway, then I am afraid this feature will have to wait for someone who is willing to do that.

Comment (Qlenglet) [reply Quarterdeck/src/main/java/com/fredboat/backend/quarterdeck/db/migrations/main/V1__InitialSchema.java]: I read the documentationa nd I can do the according work.

Thanks for your time

**Comment by napstr** [comment]
This looks even worse than before. Please do not request any more reviews from me, I'm afraid I do not have the capabilities to babysit you through this. Maybe @Frederikam is available.

**Comment by napstr** [comment]
I have asked you not to request any more reviews from me. Closing this now & reporting for abuse.

**Pull request closed without merging**